webpack.config.js

```diff
@@ -85,8 +106,19 @@ const options = {
                 ],
             },
             {
-                test: /\.(ttf|eot|woff|woff2)$/,
-                type: 'asset/resource',
+                // sync font files referenced by the css to the fonts directory
+                // the publicPath matches the path from the compiled css to the font file
+                // only looks in the fonts folder so pngs in the images folder won't get put in the fonts folder
+                test: /\.(woff|woff2)$/,
+                include: /fonts/,
+                use: {
+                    loader: 'file-loader',
+                    options: {
+                        name: '[name].[ext]',
+                        outputPath: 'fonts/',
+                        publicPath: '../fonts',
+                    },
+                },
```

Comment:
This was the update to use `asset-modules` after removing `file-loader` 
```suggestion
                type: 'asset/resource',
```